File: linguagem_dart/classes_e_objetos/hospital/paciente.dart
import 'hospital.dart';

class Paciente {
  // Atributos da classe Paciente
  String nome = "";
  late Hospital hospital;

  // Construtor da classe Paciente
  Paciente(this.nome, this.hospital) {}

  String receberAtenddimento() {
    return "O paciente $nome está recebendo atendimento no ${hospital.nome}";
  }
}

File: linguagem_java/classes_e_objetos/hospital/Paciente.java
package classes_e_objetos.hospital;

public class Paciente {
    // Atributos da classe Paciente
    String nome;
    Hospital hospital;

    // Construtor da classe Paciente
    Paciente(String nome, Hospital hospital) {
        this.nome = nome;
        this.hospital = hospital;
    }

    String receberAtendimento() {
        return "O paciente " + nome + " está recebendo atendimento no " + hospital.nome;
    }
}
